doc/introduction.rst

```diff
@@ -17,54 +17,114 @@
 .. You should have received a copy of the GNU General Public License
  along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
 
+.. Edited September 27, 2020 - [USA]TechDude
+
+User Guide
+==========
+
+This guide is a simple how-to to get the user started with enough information 
+to create a small, quick video by the end of the Quick Tutorial.  For more 
+in-depth information regarding |ops| OpenShot™, please refer to the 
+`User Manual <manual.html>`__.
+
+.. image:: images/title-ops.png
+
 Introduction
-============
+------------
 
-OpenShot Video Editor is an award-winning, open-source video editor, available on
-Linux, Mac, and Windows. OpenShot can create stunning videos, films, and animations with an
-easy-to-use interface and rich feature-set.
+Jonathan Thomas created |ops| OpenShot™ in 2008 to build a free, open-source 
```

Comment:
I would like to be omitted from this section. Keep it all about OpenShot:

OpenShot Video Editor is an award-winning, open-source video editor, available on
Linux, Mac, Windows, and Chrome OS. OpenShot can create stunning videos, films, and animations with an
easy-to-use interface and rich feature-set. We designed OpenShot to be very simple and 
quick to learn, but still offer plenty of flexibility and power.

OpenShot is available for the following operating system: (remove 64 bit reference, because we also support 32 bit windows)

Reply:
You can include a blip about the creation of OpenShot and my name on some other section or appendix. :wink: